src/sample-data/use-faker.ts

```diff
@@ -0,0 +1,9 @@
+import FakerClass from 'faker/lib'
+import { useMemo } from 'react'
+
+/**
+ * Get a new standalone (non-global) Faker instance. The returned instance will always be the
+ * same for the lifetime of the component.
+ */
+export const useFaker = (seed = 0): Faker.FakerStatic =>
+  useMemo(() => new FakerClass({ locales: require('faker/lib/locales') }), [seed])
```

Comment:
Is there a reason why this is in its own hook, and not directly in `use-sample-data`?

Reply:
It is to wrap the Faker instanciation with seed functionality in its own hook to separate concerns. However, it is not exposed by `@trampoline/cubes` (see `index.ts`). I would keep it like that, as later on, it could also handle locales loading and other options.